init.php

```diff
@@ -227,6 +227,10 @@ public static function register_eval_safe_tokens(): void {
             'wc_memberships_is_user_active_member',
         ];
 
+        if ( get_option( 'cxl_eval_safe_tokens' ) === $cxl_eval_safe_tokens ) {
+            return;
+        }
+
         update_option( 'cxl_eval_safe_tokens', $cxl_eval_safe_tokens );
 
     }
```

Comment:
I'm thinking this whole `register()` function can go away. Let's

1. register tokens manually via CLI for now
2. maybe put a Pods settings screen in front of it later

Hard-coding a list here doesn't make much sense anyway.